bookx/settings.py

```diff
@@ -124,16 +124,22 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 STATIC_URL = '/static/'
+
+if not DEBUG:
+    STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
```

Comment:
You should be able to just use this everywhere (with or without DEBUG), and point to `static` and remove the `STATICFILES_DIRS` directive.

Reply:
For some reason that didn't work for me, but STATICFILES_DIRS works. Maybe we can look at that?